scene/2d/audio_stream_player_2d.h

```diff
@@ -81,6 +81,9 @@ class AudioStreamPlayer2D : public Node2D {
 	float max_distance = 2000.0;
 	float attenuation = 1.0;
 
+	float panning_strength = 1.0f;
+	float cached_global_panning_strength;
```

Comment:
I would initialize to `1.0f` for consistency, even if it's overridden in the constructor. Better initialize twice than taking the risk of using it uninitialized. (Same in 3D.)